[2024-08-29T14:23:57.457+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_metrics.publish.customer_transactions_run scheduled__2024-08-28T00:00:00+00:00 [queued]>
[2024-08-29T14:23:57.466+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_metrics.publish.customer_transactions_run scheduled__2024-08-28T00:00:00+00:00 [queued]>
[2024-08-29T14:23:57.469+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-08-29T14:23:57.488+0000] {taskinstance.py:2214} INFO - Executing <Task(DbtRunLocalOperator): publish.customer_transactions_run> on 2024-08-28 00:00:00+00:00
[2024-08-29T14:23:57.494+0000] {standard_task_runner.py:60} INFO - Started process 61103 to run task
[2024-08-29T14:23:57.501+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'customer_metrics', 'publish.customer_transactions_run', 'scheduled__2024-08-28T00:00:00+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/customer_metrics.py', '--cfg-path', '/tmp/tmp3kdbu4_1']
[2024-08-29T14:23:57.502+0000] {standard_task_runner.py:88} INFO - Job 16: Subtask publish.customer_transactions_run
[2024-08-29T14:23:57.542+0000] {task_command.py:423} INFO - Running <TaskInstance: customer_metrics.publish.customer_transactions_run scheduled__2024-08-28T00:00:00+00:00 [running]> on host d3a9f90a4da3
[2024-08-29T14:23:57.738+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='customer_metrics' AIRFLOW_CTX_TASK_ID='publish.customer_transactions_run' AIRFLOW_CTX_EXECUTION_DATE='2024-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-28T00:00:00+00:00'
[2024-08-29T14:23:57.744+0000] {local.py:202} INFO - Cloning project to writable temp directory /tmp/tmpmc6en54w from /opt/airflow/include/dbt/fraud
[2024-08-29T14:23:57.758+0000] {config.py:253} INFO - Using user-supplied profiles.yml at /opt/airflow/include/dbt/fraud/profiles.yml
[2024-08-29T14:23:57.766+0000] {local.py:235} INFO - Trying to run the command:
 ['/home/airflow/.local/bin/dbt', 'run', '--models', 'customer_transactions', '--profiles-dir', '/opt/airflow/include/dbt/fraud', '--profile', 'fraud', '--target', 'dev']
From /tmp/tmpmc6en54w
[2024-08-29T14:23:57.770+0000] {local.py:236} INFO - Using environment variables keys: dict_keys(['AIRFLOW_CTX_DAG_OWNER', 'AIRFLOW_CTX_DAG_ID', 'AIRFLOW_CTX_TASK_ID', 'AIRFLOW_CTX_EXECUTION_DATE', 'AIRFLOW_CTX_TRY_NUMBER', 'AIRFLOW_CTX_DAG_RUN_ID'])
[2024-08-29T14:23:57.771+0000] {subprocess.py:60} INFO - Tmp dir root location: 
 /tmp
[2024-08-29T14:23:57.771+0000] {subprocess.py:73} INFO - Running command: ['/home/airflow/.local/bin/dbt', 'run', '--models', 'customer_transactions', '--profiles-dir', '/opt/airflow/include/dbt/fraud', '--profile', 'fraud', '--target', 'dev']
[2024-08-29T14:23:57.780+0000] {subprocess.py:84} INFO - Command output:
[2024-08-29T14:23:59.016+0000] {subprocess.py:94} INFO - [0m14:23:59  Running with dbt=1.8.5
[2024-08-29T14:23:59.860+0000] {subprocess.py:94} INFO - [0m14:23:59  Registered adapter: snowflake=1.8.3
[2024-08-29T14:23:59.879+0000] {subprocess.py:94} INFO - [0m14:23:59  Unable to do partial parsing because saved manifest not found. Starting full parse.
[2024-08-29T14:24:00.762+0000] {subprocess.py:94} INFO - [0m14:24:00  Found 3 models, 448 macros
[2024-08-29T14:24:00.764+0000] {subprocess.py:94} INFO - [0m14:24:00
[2024-08-29T14:24:02.228+0000] {subprocess.py:94} INFO - [0m14:24:02  Concurrency: 1 threads (target='dev')
[2024-08-29T14:24:02.229+0000] {subprocess.py:94} INFO - [0m14:24:02
[2024-08-29T14:24:02.236+0000] {subprocess.py:94} INFO - [0m14:24:02  1 of 1 START sql table model prod.customer_transactions ........................ [RUN]
[2024-08-29T14:24:03.621+0000] {subprocess.py:94} INFO - [0m14:24:03  1 of 1 OK created sql table model prod.customer_transactions ................... [[32mSUCCESS 1[0m in 1.38s]
[2024-08-29T14:24:03.625+0000] {subprocess.py:94} INFO - [0m14:24:03
[2024-08-29T14:24:03.626+0000] {subprocess.py:94} INFO - [0m14:24:03  Finished running 1 table model in 0 hours 0 minutes and 2.86 seconds (2.86s).
[2024-08-29T14:24:03.660+0000] {subprocess.py:94} INFO - [0m14:24:03
[2024-08-29T14:24:03.660+0000] {subprocess.py:94} INFO - [0m14:24:03  [32mCompleted successfully[0m
[2024-08-29T14:24:03.660+0000] {subprocess.py:94} INFO - [0m14:24:03
[2024-08-29T14:24:03.661+0000] {subprocess.py:94} INFO - [0m14:24:03  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2024-08-29T14:24:04.403+0000] {subprocess.py:98} INFO - Command exited with return code 0
[2024-08-29T14:24:04.413+0000] {local.py:138} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v12.json is above dbt-ol supported version 7. This might cause errors.
[2024-08-29T14:24:04.413+0000] {local.py:138} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/run-results/v6.json is above dbt-ol supported version 5. This might cause errors.
[2024-08-29T14:24:04.417+0000] {local.py:252} INFO - Inlets: []
[2024-08-29T14:24:04.417+0000] {local.py:253} INFO - Outlets: [Dataset(uri='snowflake://pg46124.europe-west2.gcp/AIRBYTE_DATABASE.prod.customer_transactions', extra=None)]
[2024-08-29T14:24:04.418+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-08-29T14:24:04.444+0000] {dag.py:3834} INFO - Setting next_dagrun for customer_metrics to 2024-08-29 00:00:00+00:00, run_after=2024-08-30 00:00:00+00:00
[2024-08-29T14:24:04.467+0000] {local.py:371} INFO - [0m14:24:03  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2024-08-29T14:24:04.473+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py:1469 RemovedInAirflow3Warning: Passing 'execution_date' to 'TaskInstance.xcom_push()' is deprecated.
[2024-08-29T14:24:04.491+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=customer_metrics, task_id=publish.customer_transactions_run, execution_date=20240828T000000, start_date=20240829T142357, end_date=20240829T142404
[2024-08-29T14:24:04.549+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-08-29T14:24:04.570+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
